reproject/high_level.py

```diff
@@ -63,6 +63,9 @@ def reproject(input_data, output_projection, shape_out=None, projection_type='bi
             wcs_in = WCS(input_data[1])
         else:
             wcs_in = input_data[1]
+    elif isinstance(input_data, HDUList):
+        array_in = [input_data[x].data for x in range(len(input_data))]
+        wcs_in = [WCS(input_data[x].header) for x in range(len(input_data))]
```

Comment:
Damn, this shouldn't be here at all, this was supposed to be on another branch. Clearly I didn't routed the new branch from the master...